packages/mds-policy-service/@types/models.ts

```diff
@@ -155,9 +155,15 @@ export type RatePolicy = PolicyDomainModel & { rules: RateRule[] }
 
 export type PolicyDomainCreateModel = DomainModelCreate<PolicyDomainModel>
 
+export const POLICY_TEMPLATES = ['pickup_dropoff', 'dwell_time', 'fleet_caps', 'trip_fees'] as const
+export type POLICY_TEMPLATES = typeof POLICY_TEMPLATES[number]
+
 export interface PolicyMetadataDomainModel<M extends {} = {}> {
   policy_id: UUID
+  template?: Nullable<POLICY_TEMPLATES>
   policy_metadata: Nullable<Partial<M>>
 }
 
 export type PolicyMetadataDomainCreateModel = DomainModelCreate<PolicyMetadataDomainModel>
+
+export type PolicyMetadataDomainUpdateModel<M extends {} = {}> = PolicyMetadataDomainModel<M>
```

Comment:
I'm not convinced of the need for a new type here - why not continue using `PolicyMetadataDomainCreateModel` for the update?

Reply:
Update model removed in favor of "update" column option. See 72bbf98